src/app/dashboard/components/releases-list/releases-list.component.html

```diff
@@ -0,0 +1,78 @@
+<div class="releases">
```

Comment:
Please put a breadcrumb bar above this table, to let the user navigate back to projects. The breadcrumb should have exactly the same styling as what we have when we landed in one project. (home icon + project's name)